assets/utils/commands/bump.js

```diff
@@ -0,0 +1,163 @@
+const output = require('../utility/output');
+const Input = require('../utility/input');
+const repository = require('../utility/repository');
+const { execSync } = require('child_process');
+const path = require('path');
+const process = require('process');
+const readline = require('readline');
+
+const c = output.color;
+
+async function bump(...inputList) {
+  const autoYes = inputList.includes('--yes') || inputList.includes('-y');
+  inputList = inputList.filter(arg => arg !== '--yes' && arg !== '-y');
```

Comment:
Just an idea for your next day of coolness: Maybe this could be improved by adding some configuration variables (like `autoStage`, `autoTag` and `autoPush`) that the dev can use to set his/her preferences.